src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -2510,12 +2709,29 @@ private TypeSymbolWithAnnotations ApplyConversion(BoundExpression operand, Conve
                 case ConversionKind.MethodGroup:
                     if (!fromExplicitCast)
                     {
-                        ReportNullabilityMismatchWithTargetDelegate(operandOpt.Syntax, targetType.GetDelegateType(), conversion.Method);
+                        ReportNullabilityMismatchWithTargetDelegate(node.Syntax, targetType.GetDelegateType(), conversion.Method);
                     }
                     isNullableIfReferenceType = false;
                     break;
 
                 case ConversionKind.AnonymousFunction:
+                    if (operandOpt.Kind == BoundKind.Lambda)
```

Comment:
>operandOpt [](start = 24, length = 10)

Can `operandOpt` be `null` here?

Reply:
I don't believe so.

---
In reply to: [194544279](https://github.com/dotnet/roslyn/pull/27283#discussion_r194544279) [](ancestors = 194544279)